src/pages/myGeotab/guides/mgMediaFiles.tsx

```diff
@@ -106,7 +106,7 @@ const objectModel: ReactNode = (
       </table>
     </div>
     <p>*Required when adding</p>
-    <h4>Solution Id</h4>
+    <h4>Solution id</h4>
```

Comment:
```suggestion
    <h4>Solution ID</h4>
```
Based on Geotab Written Style Guide, headings are sentence case so according to ChatGPT:
"Solution id" is not proper sentence case. In sentence case, the first letter of the first word should be capitalized, and the rest of the sentence should be in lowercase unless there are proper nouns or other terms that are typically capitalized. Additionally, "id" is an abbreviation for "identifier," and abbreviations are often capitalized. The correct sentence case would be:

"Solution ID"

Here, "Solution" is capitalized as the first word, and "ID" is capitalized because it is a common abbreviation that is typically presented in uppercase.